glossary.html

```diff
@@ -19,15 +19,31 @@
 <header id="top">
     <nav>
         <ul>
-            <p><a herf=index.html>Home</a></p>
-            <p><a herf=links.html>Links</a></p>
-            <p><a herf=glossary.html>glossary</a></p>
-            </ul>
-            </nav> </header>
```

Comment:
There are quite a few indentation problems in your code.  Please use the "Prettier Code Formatter" extension to format your code before submitting.

Also, there are many validation problems in your code.  You should validate your code and get it in valid form before submitting